test/utils.ts

```diff
@@ -47,11 +47,13 @@ export function assertValidSchema(
         const actual: any = JSON.parse(JSON.stringify(schema));
 
         // uncomment to write test files
-        // writeFileSync(
-        //     resolve(`${basePath}/${relativePath}/schema.json`),
-        //     JSON.stringify(schema, null, 4) + "\n",
-        //     "utf8"
-        // );
+        if (process.env.UPDATE_SCHEMA) {
```

Comment:
Awesome. Thank you for this. Can you add a script to the package.json to run the tests with the env var set?

Reply:
done.